spec/routes_spec.rb

```diff
@@ -17,6 +19,41 @@
           it 'method defined' do
             expect(subject.respond_to?(action)).to be_truthy
           end
+
+          describe 'RBAC' do
+            before(:all) { MiqProductFeature.seed_features }
+
+            before do
+              allow(PrivilegeCheckerService).to receive(:new).and_return(check_service)
+              allow(check_service).to receive(:valid_session?).and_return(true)
+              allow(subject).to receive(:session)
+              allow(subject).to receive(:current_user)
+              allow(subject).to receive(:params).and_return({})
+            end
+
+            let(:check_service) { double(PrivilegeCheckerService) }
+
+            ignore.key?(controller.to_s) && ignore[controller.to_s].include?(action)
+
+            it 'is enforced' do
+              pending('ignored') if ignore.key?(controller.to_s) && ignore[controller.to_s].include?(action)
```

Comment:
Think we can shorten this...

```suggestion
              pending('ignored') if ignore[controller.to_s]&.include?(action)
```